src/block.tsx

```diff
@@ -88,51 +98,58 @@ export const Block: React.FC<Block> = props => {
           const coverPosition = (1 - (page_cover_position || 0.5)) * 100;
 
           return (
-            <div className="notion">
-              {page_cover && (
-                <img
-                  src={toNotionImageUrl(page_cover)}
-                  alt={getTextContent(blockValue.properties.title)}
-                  className="notion-page-cover"
-                  style={{
-                    objectPosition: `center ${coverPosition}%`
-                  }}
-                />
-              )}
-              <div
-                className={classNames(
-                  "notion-page",
-                  !page_cover && "notion-page-offset",
-                  page_full_width && "notion-full-width",
-                  page_small_text && "notion-small-text"
-                )}
-              >
-                {page_icon && (
-                  <PageIcon
-                    className={
-                      page_cover ? "notion-page-icon-offset" : undefined
-                    }
-                    block={block}
-                    big
-                  />
-                )}
-                <div className="notion-title">
-                  {renderChildText(blockValue.properties.title)}
+            <div className="notion notion-app">
```

Comment:
these additional divs match notion's layout more closely and allow us to have the header not be part of the scrollable portion of the page (in `fullPage` mode).